<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CitizenResolutionDialog</class>
 <widget class="QDialog" name="CitizenResolutionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>543</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <widget class="QWidget" name="containerWidget" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <widget class="QToolButton" name="cancelBtn">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>348</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI Light</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QLabel" name="nameLabel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>180</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI Light</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Citizen决议投票</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QToolButton" name="okBtn">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>348</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI Light</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QToolButton" name="closeBtn">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>18</width>
       <height>18</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>400</width>
       <height>280</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QScrollArea{border:none;}</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>280</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#scrollAreaWidgetContents{background-color: rgb(229, 226, 240);}</string>
      </property>
      <widget class="QStackedWidget" name="stackedWidget_fee">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>170</y>
         <width>200</width>
         <height>140</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{color: rgb(137,129,161);
font: 11px &quot;Microsoft YaHei UI Light&quot;;}</string>
       </property>
       <property name="text">
        <string>投票账户:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>100</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{color: rgb(137,129,161);
font: 11px &quot;Microsoft YaHei UI Light&quot;;}</string>
       </property>
       <property name="text">
        <string>议题:</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="titlePlainTextEdit">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>40</y>
         <width>200</width>
         <height>100</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPlainTextEdit{background-color:white;border:none;}</string>
       </property>
      </widget>
      <widget class="QLabel" name="accountLabel">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>10</y>
         <width>140</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{font: 11px &quot;Microsoft YaHei UI Light&quot;;}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>150</y>
         <width>100</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{color: rgb(137,129,161);
font: 11px &quot;Microsoft YaHei UI Light&quot;;}</string>
       </property>
       <property name="text">
        <string>选项:</string>
       </property>
      </widget>
     </widget>
    </widget>
    <zorder>scrollArea</zorder>
    <zorder>cancelBtn</zorder>
    <zorder>nameLabel</zorder>
    <zorder>okBtn</zorder>
    <zorder>closeBtn</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
